aquarius/app/assets.py

```diff
@@ -686,6 +692,13 @@ def check_required_attributes(required_attributes, data, method):
     return None, None
 
 
+def check_no_urls_in_files(base, method):
+    if 'files' in base and 'url' in base['files']:
```

Comment:
It seems that `'url' in base['files']` will fail. `base['files']` is a list of dicts.
Maybe it should be 'url' in base['files'][0]. Since it's a list, the check should be checking each item in the list.